App.js

```diff
@@ -1,26 +1,41 @@
-import React from 'react';
-import styled from 'styled-components/native';
+import React from 'react'
+import styled from 'styled-components/native'
+import ShakeForQoute from './components/ShakeForQoute'
+import Homepage from './components/Homepage'
+import LocationStatus from './components/LocationStatus'
+// import Fortune from './components/Fortune'
+
+import { createDrawerNavigator } from '@react-navigation/drawer'
+import { NavigationContainer } from '@react-navigation/native'
 
 const Container = styled.View`
-	flex: 1;
-	background-color: papayawhip;
-	justify-content: center;
-	align-items: center;
-`;
+  flex: 1;
+  background-color: red;
+  justify-content: center;
+  align-items: center;
+  padding: 7%;
+  justify-content: space-evenly;
+  border: 5px solid black;
+`
 
 const Title = styled.Text`
-	font-size: 24px;
-	color: palevioletred;
-`;
+  font-size: 50px;
+  color: white;
+  text-align: center;
+`
 
 const App = () => {
-	return (
-		<Container>
-			<Title>This is your cool app!</Title>
-			<Title>Go to App.js and start coding</Title>
-			<Title>💅💅💅</Title>
-		</Container>
-	);
-};
+  const Drawer = createDrawerNavigator()
+  return (
+    <NavigationContainer>
+      <Drawer.Navigator initialRouteName="Homepage">
```

Comment:
aha , i see you have applied navigation and have 3 separate page-components . Cool!